cmd/minikube/cmd/delete.go

```diff
@@ -36,6 +37,16 @@ associated files.`,
 		fmt.Println("Deleting local OpenShift cluster...")
 		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 		defer api.Close()
+		host, err := api.Load(constants.MachineName)
```

Comment:
This block is same as stop.go but I think we should not put in the cluster.go because that is something coming from minikube and shouldn't disturb it.